modular_skyrat/code/datums/descriptors/descriptor_types/apparent_age.dm

```diff
@@ -0,0 +1,23 @@
+/datum/mob_descriptor/apparent_age
+	name = "apparent age"
+	replace_are = "appear to be"
+	standalone_value_descriptors = list(
+		"very young",
+		"young",
+		"adult",
+		"middle-aged",
+		"elderly",
+		)
+	comparative_value_descriptor_equivalent = "around the same age as you"
+	comparative_value_descriptors_smaller = list(
+		"slightly younger than you",
+		"younger than you",
+		"much younger than you",
+		"juvenile when compared to you",
```

Comment:
haha god i'm so glad skyrat exists to draw these people away from citadel

Reply:
ebin